evalml/pipelines/components/estimators/regressors/svm_regressor.py

```diff
@@ -42,7 +42,7 @@ class SVMRegressor(Estimator):
         ProblemTypes.TIME_SERIES_REGRESSION,
     ]"""
 
-    def __init__(self, C=1.0, kernel="rbf", gamma="scale", random_seed=0, **kwargs):
+    def __init__(self, C=1.0, kernel="rbf", gamma="auto", random_seed=0, **kwargs):
```

Comment:
Should we make this change? The perf tests only considered binary classification problems.

Reply:
This is a fair point! It's hard to say since we don't have very many regression datasets in looking glass. I'll run a few tests with what we have and see if the results are consistent or not!

Reply:
@freddyaboulton Results from the regression testing is now in the performance test doc! On the very small number of datasets we have, "auto" performs better significantly more often than "scale", so I think this change should happen.